scene/main/viewport.h

```diff
@@ -500,6 +500,9 @@ class Viewport : public Node {
 	static void _bind_methods();
 	void _validate_property(PropertyInfo &p_property) const;
 
+	void _start_drag(Window *p_window);
+	void _start_resize(SubWindowResize p_edge, Window *p_window);
```

Comment:
They could be moved from the `protected:` to the `private:` section, because `Window` is a friend class of `Viewport`.